requirements_tools/upgrade_requirements.py

```diff
@@ -105,14 +105,16 @@ def make_virtualenv(args):
         print_call(
             sys.executable, '-m', 'virtualenv', venv,
             '-p', args.python, '--never-download',
+            '--no-wheel', '--no-setuptools',
```

Comment:
`--never-download` already prevents them from downloading here so `--no-setuptools` and `--no-wheel` should be unnecessary